client/JSDebugger/configurationProvider.ts

```diff
@@ -139,13 +141,18 @@ export class MLConfigurationProvider implements vscode.DebugConfigurationProvide
                     })
                 }
             }
-            const item = await vscode.window.showQuickPick(items, { placeHolder: 'Select the request to attach to' })
-            if (!item) {
-                return vscode.window.showErrorMessage('Request not selected').then(() => {
-                    return undefined	// abort
-                })
+            if (items.length > 0) {
```

Comment:
I recommend renaming `items` to `pausedRequests`, which will also allow for getting rid of that "query for paused requests" comment.